docs/develop/howtos/search_terms_migration.md

```diff
@@ -0,0 +1,39 @@
+# Create search terms mappings
+
+This guide describes the process to create search terms mappings in the application layer.
+
+This is particularly useful when a certain search term is not supported anymore by the search engine, in favour of a new term, e.g., when a field's name is modified in a mapping.
+
+The query is parsed in the application layer and the old term is replaced with the new one before passing the query to the search engine.
+
+## Steps
+
+- Identify the search term to be mapped (e.g. `q=old_term:"value"`).
+- Identify the new, equivalent search term (e.g. `q=new_term:"value"`)
+- Add or edit the following configuration variable in your application `invenio.cfg` file:
+    ```python
+    RDM_SEARCH = {
+        "query_parser_cls": QueryParser.factory(
+            tree_transformer_factory=SearchFieldTransformer.factory(
+                mapping={
+                    "old_term": "new_term"
+                }
+            ),
+        )
+    }
+    ```
+
+## Examples
```

Comment:
Maybe add an example of a simplified query for example `authors` -> `metadata.creators....`